src/test/java/org/broadinstitute/consent/http/db/DatasetDAOTest.java

```diff
@@ -56,6 +56,30 @@
 @ExtendWith(MockitoExtension.class)
 class DatasetDAOTest extends DAOTestHelper {
 
+  @Test
+  void testFindAllDatasetStudySummariesDatasetAndStudy() {
+    Dataset dataset = insertDataset();
+    Study study = insertStudyWithProperties();
+    datasetDAO.updateStudyId(dataset.getDataSetId(), study.getStudyId());
+
+    List<DatasetStudySummary> summaries = datasetDAO.findAllDatasetStudySummaries();
+    assertFalse(summaries.isEmpty());
+    assertEquals( 1, summaries.size());
```

Comment:
If you'd like, these two assertions could be consolidated thusly (ditto throughout):
```suggestion
    assertThat(summaries, hasSize(1));
```